node/src/chain_spec.rs

```diff
@@ -124,6 +124,25 @@ pub fn development_config() -> Result<ChainSpec, String> {
                         .parse()
                         .unwrap(),
                 ],
+                // Config for Staking
+                // Currently only setup for one initial staker
+                // Make sure it works with initial-authorities as staking uses both
```

Comment:
do we have task for it ?

Reply:
Old no-longer relevant comment. removed.